change-beta/@azure-communication-react-6ebfab2b-bdaa-48d1-ba30-b371de8eea75.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "area": "feature",
+  "workstream": "Calling",
+  "comment": "stabilize blur",
```

Comment:
This is what will be in the changelog - we want to celebrate a new feature going to GA! `stabilize blur` isn't indicative that  this is a new GA feature.
Unless there's a plan to update this elsewhere, perhaps something like: `General Availability release of Background Effects - your users can now blur or replace their video background`